### DISCUSSION TITLE
Change name of custom opencrx project jar (opencrx-sample.jar)

### DISCUSSION THREAD

**Discussion by nyemiks** [opencrx/opencrx-documentation#53] [Q&A]
I have observed that when I build a custom opencrx project. A jar file named opencrx-sample.jar is generated. I now have two custom opencrx projects. I created them for different projects. 

How can I change the name of the generated jar file ?

I no longer want the generated jar file to have the name "opencrx-sample.jar". For example, I want the name to be "real-estate.jar" or any other name. I know it has to do with one of the gradle properies files. I have no idea which one.

**Comment by javac9** [comment]
Check gradle scripts, I think it's defined somewhere in them. Or obviously search the whole project for `opencrx-sample` and change it there.
